tests/01_basic.spec.tsx

```diff
@@ -19,6 +21,20 @@ test('createSlice', () => {
   expect(createSlice(slice)).toBe(slice);
 });
 
+test('createSliceWithImmer', () => {
+  const immerSlice = createSliceWithImmer({
+    name: 'counter',
+    value: {
+      count: 0
+    },
+    actions: {
+      increment: () => (prev) => prev.count += 1,
+    },
+  });
+  // returns the input
+  expect(createSlice(immerSlice)).toBe(immerSlice);
```

Comment:
Hmmm? I don't think we need to use `createSlice` after `createSliceWithImmer`.

Reply:
This was an oversight, it was meant to be `createSliceWithImmer`, thanks for catching it! I modified the test because a strict equality check shouldn't pass as the actions will be wrapped in `produce`